app/controllers/TimeController.scala

```diff
@@ -0,0 +1,175 @@
+package controllers
+
+import java.util.Calendar
+import javax.inject.Inject
+
+import com.scalableminds.util.tools.Fox
+import models.annotation.AnnotationDAO
+import models.project.ProjectDAO
+import models.task.{Task, TaskService, TaskTypeDAO}
+import models.user.time.{TimeSpan, TimeSpanDAO}
+import models.user.{User, UserDAO, UserService}
+import oxalis.security.{AuthenticatedRequest, Secured}
+import play.api.i18n.MessagesApi
+import play.api.libs.concurrent.Execution.Implicits._
+import play.api.libs.json.{JsObject, JsValue, Json}
+import play.api.mvc.AnyContent
+import play.api.i18n.{Messages, MessagesApi}
+
+import scala.concurrent.Future
+
+class TimeController @Inject()(val messagesApi: MessagesApi) extends Controller with Secured {
+
+  // REST API
+
+  //all users with working hours > 0
+  def getWorkingHoursOfAllUsers(year: Int, month: Int) = Authenticated.async { implicit request =>
+    for {
+      users <- UserDAO.findAll
+      js <- loggedTimeForUserList(users, year, month)
+    } yield {
+      Ok(js)
+    }
+  }
+
+  //list user with working hours > 0 (only one user is also possible)
+  def loggedTimeForMultipleUser(userString: String, year: Int, month: Int) = Authenticated.async { implicit request =>
+  for {
+      users <- Fox.combined(getUsersAsStringForEmail(userString.split(",").toList)) ?~> Messages("user.email.invalid")
+      js <- loggedTimeForUserList(users, year, month)
+    } yield {
+      Ok(js)
+    }
+  }
+
+  //helper methods
+
+  def loggedTimeForUserList(users: List[User], year: Int, month: Int) (implicit request: AuthenticatedRequest[AnyContent]): Future[JsValue] =  {
+    lazy val startDate = Calendar.getInstance()
+    lazy val endDate = Calendar.getInstance()
+    startDate.set(year, month - 1, startDate.getActualMinimum(Calendar.DAY_OF_MONTH), 0, 0, 0)
+    startDate.set(Calendar.MILLISECOND, 0)
+    endDate.set(year, month - 1, endDate.getActualMaximum(Calendar.DAY_OF_MONTH), 23, 59, 59)
+    endDate.set(Calendar.MILLISECOND, 999)
+
+    val filteredListByPermission = users.filter(user => request.user.isAdminOf(user))
+    val futureJsObjects = getUserWithWorkingHours(filteredListByPermission, startDate, endDate)
+
+    for {
+      jsObjectList <- Fox.sequence(futureJsObjects)
+    } yield {
+      Json.toJson(jsObjectList.flatten)
+    }
+  }
+
+  def getUserWithWorkingHours(users: List[User], startDate: Calendar, endDate: Calendar)(implicit request: AuthenticatedRequest[AnyContent]): List[Fox[JsObject]] = {
+    for {
+      user <- users
+    } yield {
+      getUserHours(user, startDate, endDate)
+    }
+  }
+
+  def getUserHours(user: User, startDate: Calendar, endDate: Calendar)(implicit request: AuthenticatedRequest[AnyContent]): Fox[JsObject] = {
+    for {
+      timeList <- TimeSpanDAO.findByUser(user, Some(startDate.getTimeInMillis), Some(endDate.getTimeInMillis))
+      timeListWithTask <- getOnlyTimeSpansWithTask(timeList)
```

Comment:
I think it would make sense to find the task for each timespan and save it with the timespan, so we don't need to request it again during json serialization